templates/javascript/package.json

```diff
@@ -9,13 +9,11 @@
 	"keywords": [
 		"javacsript",
 		"sern",
-		"discord.js",
-		"cli"
+		"discord.js"
 	],
-	"author": "EvolutionX",
 	"license": "UNLICENSED",
 	"dependencies": {
-		"@sern/handler": "^1.0.1-beta",
+		"@sern/handler": "^1.0.4-beta",
```

Comment:
Update the version according to latest changes whenever free

Reply:
will do